<?xml version='1.0' encoding='utf-8'?>
<launch>
	<arg name="world" value="empty_with_pedestrians" />
	<node name="pedestrian_tracker" output="screen" pkg="pedestrian_tracker" type="li2former_ros.py">
		<param name="weight" value="$(find pedestrian_tracker)/weight/ckpt_jrdb_ann_drow3_e40.pth" />
		<param name="model" value="Li2Former" />
		<param name="cfg" value="$(find pedestrian_tracker)/scripts/cfgs/ros_drownet.yaml" />
	</node>
	<include file="$(find ped_simulation)/launch/config.launch">
		<arg name="model" value="turtlebot3_waffle" />
		<arg name="x_pos" value="5.0" />
		<arg name="y_pos" value="2.0" />
		<arg name="z_pos" value="0.0" />
		<arg name="yaw" value="1.57" />
		<arg name="map" value="passage" />
		<arg name="rviz_file" value="sim_env.rviz" />
		<arg name="world" value="$(arg world)" />
	</include>
</launch>
